src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -79,6 +79,16 @@ - (void)commonInit {
   _scrollView.showsVerticalScrollIndicator = NO;
   _scrollView.showsHorizontalScrollIndicator = NO;
 
+  NSOperatingSystemVersion iOS9Version = {9, 0, 0};
+  NSProcessInfo *processInfo = [NSProcessInfo processInfo];
+  if ([processInfo respondsToSelector:@selector(isOperatingSystemAtLeastVersion:)] &&
+      [processInfo isOperatingSystemAtLeastVersion:iOS9Version] &&
+      [UIView
+           userInterfaceLayoutDirectionForSemanticContentAttribute:self.semanticContentAttribute] ==
```

Comment:
Should we override setSemanticContentAttribute: and make it propagate changes and invalidate state?